examples/gallery/lines/linestrings.py

```diff
@@ -0,0 +1,41 @@
+"""
+GeoPandas: Plotting lines with LineString/MultiLineString geometry
+==================================================================
+
+The :meth:`pygmt.Figure.plot` method allows us to plot geographical data
+such as lines which are stored in a :class:`geopandas.GeoDataFrame` object.
+Use :func:`geopandas.read_file` to load data from any supported OGR format
+such as a shapefile (.shp), GeoJSON (.geojson), geopackage (.gpkg), etc.
+Then, pass the :class:`geopandas.GeoDataFrame` as an argument to the
+``data`` parameter of :meth:`pygmt.Figure.plot`, and style the geometry
+using the ``pen`` parameter.
+"""
+
+# %%
+import geodatasets
+import geopandas as gpd
+import pygmt
+
+# Read shapefile data of large rivers in Europe using geopandas
+gdf = gpd.read_file(geodatasets.get_path("eea large_rivers"))
+
+# Convert object to EPSG 4326 coordinate system
+gdf = gdf.to_crs("EPSG:4326")
+print(gdf)
+
+fig = pygmt.Figure()
+
```

Comment:
I would split the code into two parts via `# %%`, to have the table showing the GeoDataFrame displayed within the code and before the image. Using `.head()` displays only the first lines of the GeoDataFrame, which should be enough to get an idea about the data. Both `gdf.head()` and `print(gdf.head())` work in the Gallery (with a slightly different appearance). But for using `gdf.head()`, in "normal" Python scripts it's only displayed when running the subsection, but not when running the complete script. So, using `print(gdf.head())`is maybe better here.

```suggestion
# Convert object to EPSG 4326 coordinate system
gdf = gdf.to_crs("EPSG:4326")
print(gdf.head())

# %%
fig = pygmt.Figure()

```